### DISCUSSION TITLE
FR: add Reload Location & customization to quick bar

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#10067] [Dashboard] [answered]
following   https://www.home-assistant.io/docs/tools/quick-bar/#a-command-is-missing request:
as title, please add the 'Reload location and customization' to the quick bar.

If at all possible, also consider adding 'Reload core config'

[discord](https://discord.com/channels/330944238910963714/460846676358332417/889844392779739136):

**Comment by bramkragten** [comment]
Core === Location and customization

**Comment by Mariusthvdb** [comment]
didn't realize the 2 were the same. still, neither is in the quickbar, so the fr remains but could be simplified to only add the 'Reload location and customization' to the quick bar, to brave it in sync with the options on /config/server_control

**Comment by spacegaier** [comment, marked as answer]
Part of https://github.com/home-assistant/frontend/pull/11072

**Comment by Mariusthvdb** [comment]
working nicely, for Customization And Themes! thanks for adding this, really cool.
